config/crd/bases/atlas.mongodb.com_atlasdeployments.yaml

```diff
@@ -55,6 +55,18 @@ spec:
               AtlasDeploymentSpec defines the desired state of AtlasDeployment
               Only one of DeploymentSpec, AdvancedDeploymentSpec and ServerlessSpec should be defined
             properties:
+              atlasRef:
+                properties:
+                  credentials:
+                    description: Credentials is the name of the secret that holds
+                      the Atlas credentials
+                    type: string
```

Comment:
Same thing as with DB User: https://github.com/mongodb/mongodb-atlas-kubernetes/pull/1769#pullrequestreview-2242716924 it should be a reference to the secret including a namespace

Reply:
Resolved as won't do